<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kaps_manager_form</class>
 <widget class="QDialog" name="kaps_manager_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>База данных вариантов решения тематических задач КАПС</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>КАПС</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="tvKaps">
          <column>
           <property name="text">
            <string>Название КАПС</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Описание</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Название каталога</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Природопользование</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>PP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ВХ</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>VX</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ДХ</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>DX</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>НГК</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>NG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ЭЭ</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>EE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ТРК</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>TR</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Тематические задачи КАПС</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QToolButton" name="tbAddTask">
            <property name="toolTip">
             <string>Добавить новую тематическую задачу</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/add.png</normaloff>:/ddoc/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbEditTask">
            <property name="toolTip">
             <string>Редактировать параметры тематической задачи</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/edit.png</normaloff>:/ddoc/edit.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbDelTask">
            <property name="toolTip">
             <string>Удалить тематическую задачу</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/delete.png</normaloff>:/ddoc/delete.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tvTasks">
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Описание</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Название каталога</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Проекты (варианты решения тематических задач)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="tbAddTaskVar">
            <property name="toolTip">
             <string>Создать новый проект (вариант решения)</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/add.png</normaloff>:/ddoc/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbEditTaskVar">
            <property name="toolTip">
             <string>Изменить параметры проекта</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/edit.png</normaloff>:/ddoc/edit.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbDelTaskVar">
            <property name="toolTip">
             <string>Удалить проект</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/delete.png</normaloff>:/ddoc/delete.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbUploadAll">
            <property name="toolTip">
             <string>Загрузить все файлы проекта из локальной ФЗ в БД</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/add_copy.png</normaloff>:/ddoc/add_copy.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbDownloadAll">
            <property name="toolTip">
             <string>Выгрузить все файлы проекта из БД</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/reload.png</normaloff>:/ddoc/reload.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tvTaskVars">
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Описание</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата создания</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Регион</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Автор</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Название каталога</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Файлы с данными для проектов</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QToolButton" name="tbAddFile">
            <property name="toolTip">
             <string>Добавить новый файл в проект</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/add.png</normaloff>:/ddoc/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbEditFile">
            <property name="toolTip">
             <string>Редактировать метаданные файла</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/edit.png</normaloff>:/ddoc/edit.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbDelFile">
            <property name="toolTip">
             <string>Удалить файл из проекта</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/delete.png</normaloff>:/ddoc/delete.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbDownloadFile">
            <property name="toolTip">
             <string>Выгрузить файл проекта в локальную ФС</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
              <normaloff>:/ddoc/orientation_icon.png</normaloff>:/ddoc/orientation_icon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tvFiles">
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Тип КА</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата и время съемки</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Регион</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>URL</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Снимки</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Исходные данные</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Промежуточные результаты</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Результаты решения задачи</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ddocs_ico/ddocs_icon_set.qrc"/>
 </resources>
 <connections/>
</ui>
